client/core/trade.go

```diff
@@ -452,6 +502,23 @@ func (t *trackedTrade) isSwappable(match *matchTracker) bool {
 	return false
 }
 
+// isActive will be true if the trade is <= OrderStatusBooked, or if any of the
+// matches are still negotiating.
+func (t *trackedTrade) isActive() bool {
+	t.mtx.RLock()
+	defer t.mtx.RUnlock()
+
+	if t.metaData.Status == order.OrderStatusBooked || t.metaData.Status == order.OrderStatusEpoch {
+		return true
+	}
+	for _, match := range t.matches {
+		if match.MetaData.Status < order.MakerRedeemed {
+			return true
```

Comment:
I agree that there's some ambiguity around the definition of "active". I wouldn't generally consider a maker match active if they are in `MakerRedeemed`, but in theory they are still waiting for the `redemption` request and will still go to `MatchComplete`, so that seems like action performed on the match = active? Will set to `< order.MatchComplete`.

Reply:
I get that for the maker, but this method, `(*trackedTrade).isActive`, doesn't consider if it's the user is maker or taker.  On the other hand in `tick()` where the side is considered, the status check with `MakerRedeemed` make senses.